setup.py

```diff
@@ -67,15 +67,15 @@ def makeExtension(extName, files=None):
 elif platform == 'ios':
     include_dirs = [
             join(kivy_ios_root, 'build', 'include'),
-            join(kivy_ios_root, 'build', 'include', 'SDL')]
+            join(kivy_ios_root, 'build', 'include', 'SDL2')]
     extra_link_args = [
         '-L', join(kivy_ios_root, 'build', 'lib'),
         '-undefined', 'dynamic_lookup']
     extensions.append(makeExtension('audiostream.platform.plat_ios',
         [join('audiostream', 'platform', 'ios_ext.m')]))
 
 elif platform == "darwin":
-    include_dirs.append('/usr/local/include/SDL')
+    include_dirs.append('/usr/local/include/SDL2')
```

Comment:
This is macOS and you'll break the `SDL`-only compatibility. I think there should be `USE_SDL2` env var to check before choosing which `SDL` version to choose i.e.:

`USE_SDL2=0` → `/usr/local/include/SDL`
`USE_SDL2=1` → `/usr/local/include/SDL2`

See the more global-ish approach in [Kivy `setup.py`](https://github.com/kivy/kivy/blob/0456eff89bccf93fa6bd8aada318246082f1b8cc/setup.py#L150), but basically it's about this:

    use_sdl2 = os.environ.get('USE_SDL2')  # value or None
    include_dirs.append('/usr/local/include/SDL2' if use_sdl2 else '/usr/local/include/SDL')
    libraries = ['SDL2', 'SDL2_mixer'] if use_sdl2 else ['SDL', 'SDL_mixer']
    etc